Node:slim base image update seems to have broken action

The node:slim base image was updated ~7 hours ago, and all first-interaction actions seem to have started failing since then. The error that I'm seeing during the container build is this:
![image](https://user-images.githubusercontent.com/9667869/96946973-da588a00-14af-11eb-9980-576faa8bb2ec.png)

I suspect it's related to these deprecation warnings prior to the base image getting changed:
![image](https://user-images.githubusercontent.com/9667869/96946995-ed6b5a00-14af-11eb-9d10-9cea49a53448.png)

Looks like the whole 15-slim series has the issue

```
~/Downloads/npm-issue                                                                                                                                                                                     06:51:02
❯ cat Dockerfile
ARG TAG

FROM node:${TAG}

COPY . .

RUN npm install --production

~/Downloads/npm-issue                                                                                                                                                                                     06:51:06
❯ docker build --build-arg TAG=slim .
Sending build context to Docker daemon  3.072kB
Step 1/4 : ARG TAG
Step 2/4 : FROM node:${TAG}
 ---> 922b09b21278
Step 3/4 : COPY . .
 ---> Using cache
 ---> 5bc600e1e54a
Step 4/4 : RUN npm install --production
 ---> Running in 94046b82e8b6
npm notice
npm notice New patch version of npm available! 7.0.3 -> 7.0.5
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.5>
npm notice Run `npm install -g npm@7.0.5` to update!
npm notice
npm ERR! Maximum call stack size exceeded

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-26T06_51_14_110Z-debug.log
The command '/bin/sh -c npm install --production' returned a non-zero code: 1

~/Downloads/npm-issue                                                                                                                                                                                     06:51:14
❯ docker build --build-arg TAG=15.0.0-slim .
Sending build context to Docker daemon  3.072kB
Step 1/4 : ARG TAG
Step 2/4 : FROM node:${TAG}
 ---> 2870b8b46426
Step 3/4 : COPY . .
 ---> Using cache
 ---> a3d7e2649929
Step 4/4 : RUN npm install --production
 ---> Running in 7696bb3404f0
npm notice
npm notice New patch version of npm available! 7.0.2 -> 7.0.5
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.5>
npm notice Run `npm install -g npm@7.0.5` to update!
npm notice
npm ERR! Maximum call stack size exceeded

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-26T06_51_17_971Z-debug.log
The command '/bin/sh -c npm install --production' returned a non-zero code: 1

~/Downloads/npm-issue                                                                                                                                                                                     06:51:18
❯ docker build --build-arg TAG=14-slim .
Sending build context to Docker daemon  3.072kB
Step 1/4 : ARG TAG
Step 2/4 : FROM node:${TAG}
 ---> c0f0d070c334
Step 3/4 : COPY . .
 ---> Using cache
 ---> 146a03f04076
Step 4/4 : RUN npm install --production
 ---> Using cache
 ---> 80086efbf336
Successfully built 80086efbf336

~/Downloads/npm-issue                                                                                                                                                                                     06:51:22
❯ docker build --build-arg TAG=12-slim .
Sending build context to Docker daemon  3.072kB
Step 1/4 : ARG TAG
Step 2/4 : FROM node:${TAG}
 ---> 5ac41a843e27
Step 3/4 : COPY . .
 ---> Using cache
 ---> 4c2f3db19ad4
Step 4/4 : RUN npm install --production
 ---> Using cache
 ---> c02cda38bb40
Successfully built c02cda38bb40
```

+1 for this issue. We use actions/first-interaction greetings for our YOLOv5 repo:
https://github.com/ultralytics/yolov5

actions/first-interaction is now failing though, i.e. https://github.com/ultralytics/yolov5/runs/1314359799?check_suite_focus=true

Full error message is:
```
Build container for action use: '/home/runner/work/_actions/actions/first-interaction/v1/Dockerfile'.
  /usr/bin/docker build -t 1e5c35:55d8dca5377d476c8f60062d5cc7586a -f "/home/runner/work/_actions/actions/first-interaction/v1/Dockerfile" "/home/runner/work/_actions/actions/first-interaction/v1"
  Sending build context to Docker daemon  180.2kB
  
  Step 1/4 : FROM node:slim
  slim: Pulling from library/node
  babf97a3f00a: Pulling fs layer
  d81eb1809d95: Pulling fs layer
  dbaab951a9fa: Pulling fs layer
  cb915c1b28b9: Pulling fs layer
  7c72cd70a544: Pulling fs layer
  cb915c1b28b9: Waiting
  7c72cd70a544: Waiting
  d81eb1809d95: Verifying Checksum
  d81eb1809d95: Download complete
  cb915c1b28b9: Verifying Checksum
  cb915c1b28b9: Download complete
  babf97a3f00a: Verifying Checksum
  babf97a3f00a: Download complete
  7c72cd70a544: Verifying Checksum
  7c72cd70a544: Download complete
  dbaab951a9fa: Verifying Checksum
  dbaab951a9fa: Download complete
  babf97a3f00a: Pull complete
  d81eb1809d95: Pull complete
  dbaab951a9fa: Pull complete
  cb915c1b28b9: Pull complete
  7c72cd70a544: Pull complete
  Digest: sha256:bf39db19ac53a635030d5db569cab912322147139fbf929c252359f3c50c4207
  Status: Downloaded newer image for node:slim
   ---> 922b09b21278
  Step 2/4 : COPY . .
   ---> 576c4002ad48
  Step 3/4 : RUN npm install --production
   ---> Running in a5708f53f895
  npm notice 
  npm notice New patch version of npm available! 7.0.3 -> 7.0.5
  npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.5>
  npm notice Run `npm install -g npm@7.0.5` to update!
  npm notice 
  npm ERR! Maximum call stack size exceeded
  
  npm ERR! A complete log of this run can be found in:
  npm ERR!     /root/.npm/_logs/2020-10-27T11_28_34_372Z-debug.log
  The command '/bin/sh -c npm install --production' returned a non-zero code: 1
  
  Warning: Docker build failed with exit code 1, back off 5.689 seconds before retry.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:slim
+FROM node:14.4.0-buster-slim
 
 COPY . .
 

